app/lab-integration/utils/lab-client.js

```diff
@@ -144,16 +150,37 @@ export class LabClient {
                 if(unknownMFL.length > 0) {
                     // post unknown MFL code to slack
                     console.warn('Unknown MFL codes', unknownMFL);
+
                 }
                 // console.log('RESULTS', response);
                 resolve(response);
             }).catch((err)=>{
-                console.error('LAB INTEGRATION ERROR:', err);
-                this.logRequestError(err,options).then((result) => {
-                    reject(err);
-                }).catch((error) => {
-                    reject(err);
-                });
+                // console.error('LAB INTEGRATION ERROR:', err);
+                if(err.message){
+                    console.log('Error code', err.statusCode);
+                    if(err.statusCode === 400){
```

Comment:
The status 400 will not always indicate that the lab cannot do cd4 test. I would look at the message too, if it contains the string that indicates lab does not do cd4 tests. Future errors might crop up silently and we will never know about it, or would be time-consuming to debug because it's not logged anywhere.